Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: help install test tests update dist publish promote
 SHELL=/bin/bash
-ECR_REGISTRY=672626379771.dkr.ecr.us-east-1.amazonaws.com
```

Comment:
I will update this after its merged, since i will likely need to make changes to this makefile anyway.   But we shouldn't hard code the account numbers here, its easier to manage if we use this method - https://github.com/MITLibraries/timdex-pipeline-lambdas/blob/995c43944db692e669f0ef91a9cd24b79087c3cb/Makefile#L3
`$(shell aws sts get-caller-identity --query Account --output text)`
and rely on the credentials to pull the value for the current account.

Reply:
Interesting. That does require that anyone running this to have the correct credentials for the stage they are trying to push to active. I agree with your suggestion here but I think we may need to talk through the workflow here as we are making subtle but important changes from when everything was in one account to splitting things across multiple accounts in terms of what will happen when commands are run and how to document that so dev expectations match reality so we don't accidentally push stuff to prod when we meant to stage them, etc.

Reply:
We need to talk about how we want the promotion process to go for sure.  You're correct @JPrevost that this would require correct credentials for anything to happen, and it will always pick the active credentials account to attempt to push to.